### DISCUSSION TITLE
Downloaded files are place in folder named with the name

### DISCUSSION THREAD

**Discussion by mathsyx69** [rogerfar/rdt-client#542] [Q&A]
For exemple when I  download an example.mkv file, I want to have directly the example.mkv on my download folder.

Actually when download files there are placed in a sub folder named by with the files names.

Is it possible to disable folder creation when download content ?

**Comment by harun911** [comment]
This is something I wonder too, I would like to have the files unpacked as is, without the creation of a folder with the name of the zip/rar file. Is there a way that we can achieve this?
